website/www/site/assets/scss/_capability-matrix.scss

```diff
@@ -305,6 +305,9 @@ nav.runner-switcher {
         border-top-right-radius: 6px;
         border-bottom: none;
         padding-left: 0 !important;
+        &:not(:last-child) {
+            border-right: none;
+        }
```

Comment:
This fixes the double-pixel border for adjacent tabs:
![image](https://user-images.githubusercontent.com/44893228/187459531-674dd8dd-8b3f-463a-9861-e30b42b67a40.png)